docs: add a FAQ on how to find the LCP image (#56779)
angular/angular: adev/src/content/guide/image-optimization.md

No, but this is on our roadmap, so stay tuned. 

If you're waiting on this feature, please upvote the Github issue [here](https://github.com/angular/angular/issues/56594).

> ### How do I find my LCP image with Chrome Dev Tools

Reviewer: Nit :)
```suggestion
### How do I find my LCP image with Chrome DevTools?
```

> Using the performance tab of the Chrome DevTools, click on the "start profiling an reload page" button. 

Reviewer: ```suggestion
Using the performance tab of the Chrome DevTools, click on the "start profiling and reload page" button on the top left. It looks like a page refresh icon.
```

> Using the performance tab of the Chrome DevTools, click on the "start profiling an reload page" button. 

Reviewer: Can we number these steps?

This will trigger a profiling snapshot of your Angular application. 

Once the profiling result is available, select "LCP" in the timings section.

> The summary of the Largest Contentful Paint will highlight the related node. 

Reviewer: ```suggestion
A summary entry should appear in the panel at the bottom. You can find the LCP element in the row for "related node".  Clicking on it will reveal the element in the Elements panel. 
```

> <img alt="LCP in the Chrome DevTools" src="assets/images/guide/image-optimization/devtools-lcp.png">

Reviewer: We should add a note somewhere that this only identifies the LCP element with the viewport size you're using to test. Developers should also use mobile emulation to identify the LCP element for smaller screens.
